src/Select/SelectInput.js

```diff
@@ -269,7 +273,7 @@ class SelectInput extends React.Component {
         <Menu
           id={`menu-${name || ''}`}
           anchorEl={this.state.anchorEl}
-          open={this.state.open}
+          open={open !== undefined ? open : this.state.open}
```

Comment:
It would be great to use the same logic as with the other components and with how React is handling the topic internally:
https://github.com/mui-org/material-ui/blob/33d0954b8626528fda6eeaa8cc39216ccc3117dd/src/internal/SwitchBase.js#L41

Reply:
Thanks, ok, working on it, adding tests

```js
isControlled = this.props.open !== undefined;
```